src/engine/DebugOverlays.tscn

```diff
@@ -209,12 +209,12 @@ __meta__ = {
 "_editor_description_": "PLACEHOLDER"
 }
 
-[connection signal="Closed" from="PerformanceMetrics" to="." method="OnPerformanceMetricsToggled"]
+[connection signal="Cancelled" from="PerformanceMetrics" to="." method="OnPerformanceMetricsToggled"]
 [connection signal="toggled" from="DebugPanel/VBoxContainer/Checkboxes/FPSCheckBox" to="." method="OnFpsCheckBoxToggled"]
 [connection signal="toggled" from="DebugPanel/VBoxContainer/Checkboxes/PerformanceMetricsCheckBox" to="." method="OnPerformanceMetricsCheckBoxToggled"]
 [connection signal="toggled" from="DebugPanel/VBoxContainer/Checkboxes/CollisionShapeCheckBox" to="." method="OnCollisionShapeCheckBoxToggled"]
 [connection signal="toggled" from="DebugPanel/VBoxContainer/Checkboxes/EntityLabelCheckBox" to="." method="OnEntityLabelCheckBoxToggled"]
 [connection signal="toggled" from="DebugPanel/VBoxContainer/Checkboxes/InspectorCheckBox" to="." method="OnInspectorCheckBoxToggled"]
 [connection signal="pressed" from="DebugPanel/VBoxContainer/Button" to="." method="OnDumpSceneTreeButtonPressed"]
 [connection signal="value_changed" from="DebugPanel/VBoxContainer/VBoxContainer/RigiditySlider" to="." method="OnTransparencySliderValueChanged"]
-[connection signal="Closed" from="Inspector" to="." method="OnInspectorToggled"]
+[connection signal="Cancelled" from="Inspector" to="." method="OnInspectorToggled"]
```

Comment:
Actually testing this in-game this doesn't seem to work anymore... I'll go through all of the closed vs. cancelled signals in this PR and try to figure out which should be used. Hopefully I won't make mistakes, but I'll push my changes once done to give other people a chance to check my work.